sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -52,6 +93,28 @@ export interface AspectSentiment {
 
 export { AzureKeyCredential }
 
+// @public
+export type BeginAnalyzeHealthcareOperationState = PollOperationState<PaginatedHealthcareEntities>;
+
+// @public
+export interface BeginAnalyzeHealthcareOptions {
+    // (undocumented)
+    health?: HealthcareJobOptions;
```

Comment:
I don't really understand `BeginAnalyzeHealthcareOptions` and `BeginAnalyzeOptions` having custom subkeys that are just copies of `TextAnalyticsOperationOptions` with no customization.

I would expect this to instead look like

```ts
export interface BeginAnalyzeHealthcareOptions  extends TextAnalyticsOperationOptions, PollingOptions { }
```

or possibly

```ts
export type BeginAnalyzeHealthcareOptions = TextAnalyticsOperationOptions &  PollingOptions
```

Reply:
I think you are discussing two things here:
- `analyze` key is typed as  `AnalyzeJobOptions` which is just a synonym to `TextAnalyticsOperationOptions`, so why not type it as `TextAnalyticsOperationOptions` instead? I think having a distinct relevant name makes a better user experience with intellisense and it makes it easier to extend this type in the future.
-  Why not flatten `BeginAnalyzeOptions`? I think keeping the options type as is makes it clear to the user which options are used to configure the analyze job itself and which are used to configure the poller.